schemas/appeal.schema.json

```diff
@@ -0,0 +1,555 @@
+    "caseReference": {
+      "type": "string",
+      "examples": ["7002471"],
+      "description": "External case identifier"
+    },
+    "caseStatus": {
+      "type": "string",
+      "enum": [
+        "assign_case_officer",
+        "validation",
+        "ready_to_start",
+        "lpa_questionnaire",
+        "issue_determination",
+        "complete",
+        "invalid",
+        "closed",
+        "withdrawn",
+        "awaiting_transfer",
+        "transferred"
+      ],
+      "description": "The processing status for the appeal"
+    },
+    "caseTypeCode": {
+      "type": "string",
+      "enum": ["C", "D", "F", "G", "H", "L", "Q", "S", "V", "W", "X", "Y", "Z"],
+      "description": "The internal code for an appeal type, e.g. D (Householder)"
+    },
+    "caseProcedure": {
+      "type": "string",
+      "enum": ["written", "hearing", "inquiry"],
+      "description": "The type of procedure for the appeal"
+    },
+    "lpaCode": {
+      "type": "string",
+      "examples": ["Q9999"],
+      "description": "A unique identifier for the Local Planning Authority",
+      "$comment": "Uses the internal PINS code identifying LPAs"
+    },
+    "caseOfficerId": {
+      "type": ["string", "null"],
+      "description": "Unique identifier for the case officer assigned to the case",
+      "$comment": "This is the user ID in Active Directory"
+    },
+    "inspectorId": {
+      "type": ["string", "null"],
+      "description": "Unique identifier for the inspector assigned to the case",
+      "$comment": "This is the user ID in Active Directory"
+    },
+    "allocationLevel": {
+      "type": ["string", "null"],
+      "enum": ["A", "B", "C", "D", "E", "F", "G", "H"],
+      "description": "A level used for allocation purposes"
+    },
+    "allocationBand": {
+      "type": ["number", "null"],
+      "enum": [1, 2, 3],
+      "description": "A band used for allocation purposes",
+      "$comment": "Linked to the level, e.g. 1[E,F,G,H], 2[C,D], 3[A,B]"
+    },
+    "caseSpecialisms": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "examples": [
+        "Access",
+        "Listed building and enforcement",
+        "Roads and traffics",
+        "Natural heritage",
+        "Schedule 1"
+      ],
+      "description": "A list of specialisms for allocation purposes"
+    },
+    "caseSubmittedDate": {
+      "type": "string",
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The date the appeal was submitted by the appellant"
+    },
+    "caseCreatedDate": {
+      "type": "string",
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The date the appeal was received"
+    },
+    "caseUpdatedDate": {
+      "type": "string",
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The date the appeal was last updated in the back-office"
+    },
```

Comment:
Could this be `null` when first created, or does `caseCreatedDate=caseUpdatedDate` at that point?

Reply:
Yes, it defaults to current datetime in the database, so it will be set automatically on the first write.